EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -1018,6 +1038,8 @@
 				OTHER_CFLAGS = (
 					"-fobjc-arc",
 					"-fobjc-arc-exceptions",
+					"-DNS_BLOCK_ASSERTIONS=1",
```

Comment:
As @bootstraponline stated, the issue seems to be that Carthage does not provide direct control over the build flags. @bootstraponline, is there a way by which we could add these parameters in any other way, using a custom script or the scheme or so.